datameta/api/metadatasets.py

```diff
@@ -220,7 +220,7 @@ def replace_metadatasets(request: Request) -> SubmissionResponse:
 
     if missing_msets:
         messages, entities = zip(*missing_msets)
-        raise errors.get_validation_error(messages=list(messages), entities=list(entities))
+        raise errors.get_validation_error(messages=list(messages), entities=list(entities))  # @lkuchenb: any idea why these list-casts are necessary to please mypy? they're not required in e.g validation.py:63
```

Comment:
`zip()` returns tuples, not lists

Reply:
that i know :P but why does it not complain in validation.py, line 63, which also uses zip-derived values for the arguments?

Reply:
No idea. But we should probably not be so strict about the argument type in `errors.get_validation_error` and relax it to `Iterable`/